src/api/squirrel.c

```diff
@@ -1447,7 +1447,7 @@ static void squirrel_open_builtins(HSQUIRRELVM vm)
     sq_poptop(vm);
 }
 
-static void initAPI(tic_core* core)
+static void initLuaAPI(tic_core* core)
```

Comment:
It looks like the function was accidentally renamed here, there should be no mention of Lua in the Squirrel module.